packages/compass-query-history/src/stores/favorite-list-store.js

```diff
@@ -78,7 +81,11 @@ const configureStore = (options = {}) => {
     getInitialState() {
       return {
         items: new FavoriteQueryCollection(),
-        current: null
+        current: null,
+        currentHost:
+          options.dataProvider?.dataProvider
```

Comment:
why is it called `dataProvider` and not `dataService` here? :)

Reply:
My only guess is that someone though that `dataProvider` is a more abstract / not related to Compass name than `dataService` to have it as a public interface on these plugins for cloud usage 🤷‍♂️ There are a few places where it's used, some other plugins too:

<img width="465" alt="image" src="https://user-images.githubusercontent.com/5036933/173540158-1d45cede-52fd-4fde-923a-137da20dda41.png">

Reply:
I mean .. yeah, i guess not for this PR then